packages/shared/pkg/limit/gcloud.go

```diff
@@ -21,24 +19,3 @@ func (l *Limiter) GCloudMaxTasks() int {
 
 	return maxTasks
 }
-
-func (l *Limiter) GCloudCmdLimits(path string) []string {
-	maxCPU, flagErr := l.featureFlags.IntFlag(featureflags.GcloudMaxCPUQuota, path)
-	if flagErr != nil {
-		zap.L().Warn("soft failing during gcloud cmd limits feature flag receive", zap.Error(flagErr), zap.Int("maxCPU", maxCPU))
-	}
-
-	maxMemory, flagErr := l.featureFlags.IntFlag(featureflags.GcloudMaxMemoryLimitMiB, path)
```

Comment:
can we clean up the old flags + `GcloudConcurrentUploadLimit` in `Limiter` (with the unnecessary goroutine)?